gateway/mw_request_size_limit_test.go

```diff
@@ -13,14 +13,30 @@ func TestRequestSizeLimitGlobalSizeLimit(t *testing.T) {
 	ts := StartTest(nil)
 	defer ts.Close()
 
-	ts.Gw.BuildAndLoadAPI(func(spec *APISpec) {
+	api := ts.Gw.BuildAndLoadAPI(func(spec *APISpec) {
 		UpdateAPIVersion(spec, "v1", func(v *apidef.VersionInfo) {
 			v.GlobalSizeLimit = 1024
 		})
-	})
+	})[0]
 
 	_, _ = ts.Run(t, []test.TestCase{
 		{Method: "POST", Path: "/sample/", Data: strings.Repeat("a", 1024), Code: http.StatusOK},
 		{Method: "POST", Path: "/sample/", Data: strings.Repeat("a", 1025), Code: http.StatusBadRequest},
 	}...)
+
+	t.Run("endpoint level", func(t *testing.T) {
```

Comment:
since we're adding endpoint level tests here, shall we rename the test to `TestRequestSizeLimit` from `TestRequestSizeLimitGlobalSizeLimit`

Reply:
done